src/AccelerometerThread.h

```diff
@@ -5,6 +5,7 @@
 #include "power.h"
 
 #include <Adafruit_LIS3DH.h>
+#include <Adafruit_LSM6DS3TRC.h>
```

Comment:
should there be a guard here, e.g. `#ifdef UNPHONE` so that devices that don't have that model of accel don't get the dependency?

Reply:
Well maybe there should, but seems like the others didn't get one, and I was just tryin' to fit in...

Reply:
In fact seems like all the sensors need to get defined in a platformio.ini somewhere outside of board variants... Building is failing for all the other boards! I'll be able to fix this later today

Reply:
Of course! It needs to be like this so that folks can add a sensor after compilation and it just works...

Reply:
We definitely want this to be a non-gated peripheral option, especially since it's adafruit. Let's make it globallly available